README.md

```diff
@@ -829,6 +829,7 @@ Flags:
 - :nerd_face: `--all-platforms`: Push content for all platforms
 - :nerd_face: `--sign`: Sign the image (none|cosign). See [`docs/cosign.md`](./docs/cosign.md) for details.
 - :nerd_face: `--cosign-key`: Path to the private key file, KMS, URI or Kubernetes Secret for `--sign=cosign`
+- :nerd_face: `--allow-non-distributable-blobs`: Allow pushing images with non-distributable blobs
```

Comment:
`dockerd` has `--allow-nondistributable-artifacts`, so maybe s/artifacts/blobs/ ?

Reply:
Sure

Reply:
So, can we s/blobs/artifacts/ for nerdctl?

Reply:
Done